test/template/www/js/scenarioRestart.js

```diff
@@ -36,7 +36,8 @@ var app = {
         window.codePush.restartApplication(
             function () {
                 /* success */
-                app.sendTestMessage("RESTART_SUCCEEDED");
+                // does not always send...rely on RESTART_FAILED to provide failure state instead of checking for RESTART_SUCCEEDED
+                // app.sendTestMessage("RESTART_SUCCEEDED");
```

Comment:
Can you check that its working? It might be removed in WKWebView. https://xhr.spec.whatwg.org/#synchronous-flag

> Synchronous XMLHttpRequest outside of workers is in the process of being removed from the web platform as it has detrimental effects to the end user's experience. (This is a long process that takes many years.) Developers must not pass false for the async argument when entry settings object's global object is a Window object. User agents are strongly encouraged to warn about such usage in developer tools and may experiment with throwing an InvalidAccessError exception when it occurs.